src/inmanta/deploy/scheduler.py

```diff
@@ -914,9 +921,7 @@ async def _new_version(
             self._timer_manager.remove_timers(deleted)
             # Install timers for initial up-to-date resources. They are up-to-date now,
             # but we want to make sure we periodically repair them.
-            self._timer_manager.update_timers(
-                up_to_date_resources | (transitive_unblocked - self._state.dirty), are_compliant=True
-            )
+            self._timer_manager.update_timers(up_to_date_resources | (transitive_unblocked - self._state.dirty))
```

Comment:
I think we'll have to include `resources_with_reset_requires` here (or at least the subset that we actually updated) to cover transitions from `TRANSIENT` to `NO` (ensure repair timer is replaced with deploy timer if not compliant).
```suggestion
            self._timer_manager.update_timers(up_to_date_resources | (transitive_unblocked | resources_with_reset_requires - self._state.dirty))
```

That or add them to the dirty set when we unblock them (line 909).